cdc/sink/dmlsink/mq/dispatcher/topic/expression.go

```diff
@@ -14,9 +14,9 @@
 package topic
 
 import (
-	"regexp"
-
 	"github.com/pingcap/tiflow/pkg/errors"
+	"regexp"
```

Comment:
Keep the import packages like:
```go
import (
    "regexp"
    "strings"
  
    "github.com/pingcap/tiflow/pkg/errors"
)
```
The internal packages should be in the above of all imported packages.

Reply:
fixed